### PR TITLE
Feature/aaa fixup release

### PR DESCRIPTION
**Pull request by robert-w-gries**
* Remove vshell commands from tests
* Remove platform check that failed for non-3k and non-9k platforms
* Fix `authentication_login` test that rotated values the incorrect way

Tests passing all platforms:

* `cisco_aaa_group_tacacs`
* `cisco_aaa_authentication_login`
* `cisco_aaa_login_cfg_svc`
* `cisco_aaa_login_exec`

### CODE REVIEW

File: tests/beaker_tests/lib/utilitylib.rb [hunk 1]
```diff
@@ -268,7 +268,7 @@ def test_resource(tests, id, state=false)
 # Wrapper for config pattern-match tests
 def test_show_cmd(tests, id, state=false)
   stepinfo = format_stepinfo(tests, id, 'SHOW CMD')
-  show_cmd = get_vshell_cmd(tests[:show_cmd])
+  show_cmd = get_vshell_cmd(tests[id][:show_cmd])
```
Comment (chrisvanheuveln) [inline tests/beaker_tests/lib/utilitylib.rb]: :+1: on your changes as is; just wondering if we can take this opportunity to get rid of `test_show_cmd` altogether at this point? Looks like only one test might actually still be using this and it's a aaa test...

`grep  test_show_cmd tests/* -r`
`grep grep show_pattern tests/* -r`

**Comment by chburket** [comment]
:+1:

File: tests/beaker_tests/lib/utilitylib.rb (hunk 1 above)
Comment (robert-w-gries) [inline tests/beaker_tests/lib/utilitylib.rb]: Done

**Comment by robert-w-gries** [comment]
Is it preferred to have the doc updates in this PR or in a separate one?

File: tests/beaker_tests/cisco_aaa_authentication_login/test_aaaauthelogin_nondefaults.rb [hunk 2]
```diff
@@ -97,12 +96,12 @@
 
     tests[id][:code] = [0, 2]
```
Comment (mikewiebe) [inline tests/beaker_tests/cisco_aaa_authentication_login/test_aaaauthelogin_nondefaults.rb]: I know this is not part of your changes, but I would expect the exit code to be 2 each time for this test if the test is cleaning up properly in the setup for the test.

**Comment by chrisvanheuveln** [comment]
doc updates: Your call. I like smaller reviews :-)

File: tests/beaker_tests/cisco_aaa_authentication_login/test_aaaauthelogin_nondefaults.rb (hunk 2 above)
Comment (robert-w-gries) [inline tests/beaker_tests/cisco_aaa_authentication_login/test_aaaauthelogin_nondefaults.rb]: Good catch.  Test still passes with the change

**Comment by mikewiebe** [comment]
:+1:

**Pull request merged**